_rules/lists-children-follow-context-model-c6f8a9.md

```diff
@@ -0,0 +1,130 @@
+---
+id: c6f8a9
+name: Child elements of list(s) follow context model
+rule_type: atomic
+description: |
+  This rule checks that child elements of a list follow context model
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+acknowledgments:
+  authors:
+    - Jey Nandakumar
+---
+
+## Applicability
+
+The rule applies to every `li`, `dd` and `dt` HTML elements where the [semantic role][] is the same as the [implicit semantic role][].
+
+## Expectation
+
+Each target element:
+
+- is a [palpable content][], and;
+- is contained by a node which follows the [flow content model][], and;
+- has an [explicit semantic role][] or [implicit semantic role][] that that is an allowed role by the owner element.
```

Comment:
This still has the target element as subject. By applicability, the semantic role is the implicit role.